### ISSUE TITLE
AWS: error when trying to scale a nodegroup

### ISSUE THREAD

**Issue by lbogdan** [pharmer/pharmer#388]
I have created an AWS cluster with a nodegroup with 1 node, everything worked fine. Now I tried to scale the nodegroup to 2 nodes, but when I do `pharmer apply` I get this:
```
$ pharmer --v=5 apply a1
I0215 13:14:28.945323   21176 logs.go:19] FLAG: --alsologtostderr="false"
I0215 13:14:28.946323   21176 logs.go:19] FLAG: --analytics="true"
I0215 13:14:28.947323   21176 logs.go:19] FLAG: --config-file=""
I0215 13:14:28.947323   21176 logs.go:19] FLAG: --dry-run="false"
I0215 13:14:28.947323   21176 logs.go:19] FLAG: --env="prod"
I0215 13:14:28.947323   21176 logs.go:19] FLAG: --help="false"
I0215 13:14:28.947323   21176 logs.go:19] FLAG: --log_backtrace_at=":0"
I0215 13:14:28.947323   21176 logs.go:19] FLAG: --log_dir=""
I0215 13:14:28.947323   21176 logs.go:19] FLAG: --logtostderr="true"
I0215 13:14:28.947323   21176 logs.go:19] FLAG: --stderrthreshold="2"
I0215 13:14:28.947323   21176 logs.go:19] FLAG: --v="5"
I0215 13:14:28.948323   21176 logs.go:19] FLAG: --vmodule=""
I0215 13:14:31.611475   21176 cloud.go:135] Ubuntu image with ami-1c45e273 for /dev/sda1 detected
[upgrade/versions] Cluster version: v1.9.3
I0215 13:14:32.193509   21176 upm.go:306] Upgrading server ip-172-20-0-9.eu-central-1.compute.internal using `sh -c 'echo "#!/bin/bash" > /usr/bin/pharmer.sh; echo "set -xeou pipefail" >> /usr/bin/pharmer.sh; echo "export DEBIAN_FRONTEND=noninteractive" >> /usr/bin/pharmer.sh; echo "export DEBCONF_NONINTERACTIVE_SEEN=true" >> /usr/bin/pharmer.sh; echo "" >> /usr/bin/pharmer.sh; echo "apt-get update" >> /usr/bin/pharmer.sh; echo "pre-k check master-status --timeout=-1s --kubeconfig=/etc/kubernetes/admin.conf" >> /usr/bin/pharmer.sh; echo "kubeadm upgrade apply 1.9.3 -y" >> /usr/bin/pharmer.sh; chmod +x /usr/bin/pharmer.sh; nohup /usr/bin/pharmer.sh >> /var/log/pharmer.log 2>&1 &'`
I0215 13:14:32.745540   21176 ssh.go:59] $  sh: 1:
I0215 13:14:32.746540   21176 ssh.go:59] $  cannot create /usr/bin/pharmer.sh: Permission denied
sh: 1: cannot create /usr/bin/pharmer.sh: Permission denied
sh: 1: cannot create /usr/bin/pharmer.sh: Permission denied
sh: 1:
I0215 13:14:32.791543   21176 ssh.go:59] $  cannot create /usr/bin/pharmer.sh: Permission denied
sh: 1: cannot create /usr/bin/pharmer.sh: Permission denied
sh: 1: cannot create /usr/bin/pharmer.sh: Permission denied
sh: 1: cannot create /usr/bin/pharmer.sh: Permission denied
sh: 1: cannot create /usr/bin/pharmer.sh: Permission denied
chmod: cannot access '/usr/bin/pharmer.sh': No such file or directory
I0215 13:14:33.249569   21176 cloud.go:1379] Created node configuration{

} AlreadyExists: Launch Configuration by this name already exists - A launch configuration already exists with the name a1-t2-micro-v1518652515022332200
        status code: 400, request id: 6596683a-1241-11e8-9446-39aefc727111
F0215 13:14:33.249569   21176 apply.go:34]
Caused By:
AlreadyExists: Launch Configuration by this name already exists - A launch configuration already exists with the name a1-t2-micro-v1518652515022332200
        status code: 400, request id: 6596683a-1241-11e8-9446-39aefc727111
Stack trace:
C:/Users/Bogdan/Documents/work/go/src/github.com/pharmer/pharmer/cloud/providers/aws/apply.go:669             (*ClusterManager).applyUpgrade
C:/Users/Bogdan/Documents/work/go/src/github.com/pharmer/pharmer/cloud/providers/aws/apply.go:62              (*ClusterManager).Apply
C:/Users/Bogdan/Documents/work/go/src/github.com/pharmer/pharmer/cloud/lib.go:234                             Apply
C:/Users/Bogdan/Documents/work/go/src/github.com/pharmer/pharmer/cloud/cmds/apply.go:32                       NewCmdApply.func1
C:/Users/Bogdan/Documents/work/go/src/github.com/pharmer/pharmer/vendor/github.com/spf13/cobra/command.go:702 (*Command).execute
C:/Users/Bogdan/Documents/work/go/src/github.com/pharmer/pharmer/vendor/github.com/spf13/cobra/command.go:783 (*Command).ExecuteC
C:/Users/Bogdan/Documents/work/go/src/github.com/pharmer/pharmer/vendor/github.com/spf13/cobra/command.go:736 (*Command).Execute
C:/Users/Bogdan/Documents/work/go/src/github.com/pharmer/pharmer/main.go:18                                   main
C:/Go/src/runtime/proc.go:195                                                                                 main
C:/Go/src/runtime/asm_amd64.s:2337                                                                            goexit
$ _
```

**Comment by tamalsaha** [comment]
https://github.com/pharmer/pharmer/pull/389

**Comment by lbogdan** [comment]
After investigating this, I found the root cause: I had `kubernetesVersion` set to `1.9.3` but `pharmer` expects it to be `v1.9.3` (although the initial deployment was fine). So I see two solutions:
- either validate the version and error if it doesn't have the `v` prefix;
- or ignore the 'v' prefix in the [version check](https://github.com/pharmer/pharmer/blob/master/cloud/version.go#L141), so that both `v1.9.3` and `1.9.3` work.

**Comment by tamalsaha** [comment]
This should be fixed by #390. You have to build the binary from git repo. Please reopen if you are still facing this issue.

**Comment by lbogdan** [comment]
I can confirm it now works, thanks!
